docs/content/posts/2023-05-19-release-1.17.1.md

```diff
@@ -0,0 +1,195 @@
+---
+title:  "Apache Flink 1.17.1 Release Announcement"
+date: "2023-05-19T13:00:00.000Z"
+aliases:
+- /news/2023/05/19/release-1.17.1.html
+authors:
+- reswqa:
+  name: "Weijie Guo"
+  twitter: "WeijieGuo12"
+---
+
+The Apache Flink Community is pleased to announce the first bug fix release of the Flink 1.17 series.
+
+This release includes 75 bug fixes, vulnerability fixes, and minor improvements for Flink 1.17.
+Below you will find a list of all bugfixes and improvements (excluding improvements to the build infrastructure and build stability). For a complete list of all changes see:
+[JIRA](https://issues.apache.org/jira/secure/ReleaseNote.jspa?projectId=12315522&version=12352886).
+
+We highly recommend all users upgrade to Flink 1.17.1.
+
+# Release Artifacts
+
+## Maven Dependencies
+
+```xml
+<dependency>
+  <groupId>org.apache.flink</groupId>
+  <artifactId>flink-java</artifactId>
+  <version>1.17.1</version>
+</dependency>
+<dependency>
+  <groupId>org.apache.flink</groupId>
+  <artifactId>flink-streaming-java</artifactId>
+  <version>1.17.1</version>
+</dependency>
+<dependency>
+  <groupId>org.apache.flink</groupId>
+  <artifactId>flink-clients</artifactId>
+  <version>1.17.1</version>
+</dependency>
+```
+
+## Binaries
+
+You can find the binaries on the updated [Downloads page]({{< relref "downloads" >}}).
+
+## Docker Images
+
+* [library/flink](https://hub.docker.com/_/flink?tab=tags&page=1&name=1.17.1) (official images)
```

Comment:
The link is not correct, please change to `https://hub.docker.com/_/flink/tags?page=1&name=1.17.1`